src/StateService/StatePlugin.cs

```diff
@@ -0,0 +1,57 @@
+using Akka.Actor;
+using Neo.IO.Caching;
+using Neo.Network.P2P.Payloads;
+using Neo.Persistence;
+using Neo.Plugins.StateService.Storage;
+using System.Collections.Generic;
+using System.Linq;
+using static Neo.Ledger.Blockchain;
+
+namespace Neo.Plugins.StateService
+{
+    public class StatePlugin : Plugin, IPersistencePlugin
+    {
+        public const string StatePayloadCategory = "StateService";
+        public ActorSystem ActorSystem { get; } = ActorSystem.Create(nameof(StatePlugin));
```

Comment:
You can use `Plugin.System.ActorSystem` instead of creating a new one.

Reply:
Done.